components/match2/wikis/osu/get_match_group_copy_paste_wiki.lua

```diff
@@ -0,0 +1,82 @@
+---
+-- @Liquipedia
+-- wiki=osu
+-- page=Module:GetMatchGroupCopyPaste/wiki
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+
+local BaseCopyPaste = Lua.import('Module:GetMatchGroupCopyPaste/wiki/Base')
+
+---@class OsuMatch2CopyPaste: Match2CopyPasteBase
+local WikiCopyPaste = Class.new(BaseCopyPaste)
+
+local INDENT = BaseCopyPaste.Indent
+local VETOES = {
+	[1] = {'ban', 'ban', 'ban', 'decider'},
+	[2] = {'ban', 'ban', 'pick', 'ban'},
+	[3] = {'ban', 'pick', 'ban', 'decider'},
+	[5] = {'ban', 'pick', 'pick', 'decider'},
+	[7] = {'ban', 'pick', 'pick', 'pick', 'decider'},
+	[9] = {'ban', 'pick', 'pick', 'pick', 'pick', 'decider'},
+	[11] = {'ban', 'pick', 'pick', 'pick', 'pick', 'pick', 'decider'},
+	[13] = {'ban', 'pick', 'pick', 'pick', 'pick', 'pick', 'pick', 'decider'},
```

Comment:
Are you sure these are correct? 

1,2,3 and 5 indicate a map pool of size 7
The rest indicate a map pool that's the number+2

Reply:
Have asked them,  they told me 1-3 is not used anymore but for 5 and above those are norms (e.g. 5 is so that each team picks 2 map then 1 decider for a 5 map veto) I've removed 1-3

Reply:
So best of 5 has a 7 map pool?
and a Best of 7 has a 9 map pool?

Reply:
there are still Bo1/2/3 matches, but those are now runs without a veto (map ban/pick) anymore, from Bo5 onward where matches start having Ban/Picks  

e.g. for a Bo5 

![image](https://github.com/user-attachments/assets/6f9f882f-9ec3-4eca-81e7-d141a8106a83)

Reply:
that is so strange, okay